addons/shortcodes.php

```diff
@@ -171,7 +183,13 @@ function shortcode_attendees( $attr ) {
 							$first = get_post_meta( $attendee_id, 'tix_first_name', true );
 							$last = get_post_meta( $attendee_id, 'tix_last_name', true );
 
-							echo get_avatar( get_post_meta( $attendee_id, 'tix_email', true ) );
+							// Avatar placeholders
+							echo $this->get_avatar_placeholder( get_post_meta( $attendee_id, 'tix_email', true ) );
```

Comment:
This should be escaped late, but since we need HTML we can use `wp_kses()`. If we just pass an array `div`, then it should be performant.

https://vip.wordpress.com/documentation/vip/best-practices/security/validating-sanitizing-escaping/#always-escape-late

https://www.tollmanz.com/wp-kses-performance/